modules/storage-persistent-storage-selinuxChangePolicy-namespace.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+//* microshift_storage/understanding-persistent-storage-microshift.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="using_selinuxChangePolicy_namespace_{context}"]
+= Changing seLinuxChangePolicy at the namespace level
+
+After applying the desired setting for `seLinuxChangePolicy` at the namespace level, all subsequently created pods, and restarted pods, in that namespace inherit the setting. However, if desired, you can override the inherited `seLinuxChangePolicy` setting for individual pods. Setting `seLinuxChangePolicy` at the pod level overrides inheritance from the namespace level setting for that pod.
+
+.Prerequisites
+
+* Logged in to a running {product-title} cluster with administrator privileges.
+
+* Access to the {product-title} console.
+
+* For either ReadWriteOnce (RWO) or ReadWriteMany (RWX) volumes, to set `seLinuxChangePolicy` you must enable Feature Gates. For information about enabling Feature Gates, see Section _Enabling features using feature gates_.
+
+
+.Procedure
+
+To set `SELinuxChangePolicy` per namespace:
+
+. Select the desired namespace:
+
+.. Click *Administration* > *Namespaces*. 
+
+.. On the *Namespaces* page, click the desired namespace. The *Namespace details* page appears.
+
+. Add the `seLinuxChangePolicy` label to the namespace:
+
+.. On the *Namespace details* page, next to *Labels*, click *Edit*.
+
+.. In the *Edit labels* dialog, add the label `storage.openshift.io/selinux-change-policy` and set it equal to either:
++
+* `MountOption`: Specifies avoiding recursive relabeling of all files by attempting to mount the volume with the correct SELinux label directly using the -o context mount option, thus helping to avoid pod timeout problems. This value can only be applied to ReadWriteOncePod (RWOP) volumes and is the default behavior.
+* `Recursive`: Specifies recursively relabeling all files on all pod volumes to the appropriate SELinux context. This value can be applied to RWOP, RWO, or RWX volumes; however, if you are applying it to RWO or RWX volumes, you must enable Feature Gates (see the preceding _Prerequistes_ section). It is the default behavior for RWO and RWX volumes.
+
+.. Click *Save*.
+
+.Verification
+Start up a pod in the previously edited namespace and observe that the parameter `spec.securityContext.seLinuxChangePolicy` contains the value that you set for the namespace.
+
+.Example pod YAML file showing `seLinuxChangePolicy` setting
+[source,yaml]
+----
+securityContext:
+    seLinuxOptions:
+      level: 's0:c27,c19'
+    runAsNonRoot: true
+    fsGroup: 1000740000
+    seccompProfile:
+      type: RuntimeDefault
+    seLinuxChangePolicy: MountOption <1>
```

Comment:
It would be better to show "Recursive" in the example, as it's available in all clusters.

Reply:
Fixed in latest revision.